src/browser/BrowserCodeReader.ts

```diff
@@ -237,16 +240,26 @@ export default class BrowserCodeReader {
         }
     }
 
-    private decodeOnceWithDelay(resolve: (result: Result) => any, reject: (error: any) => any): void {
-        this.timeoutHandler = window.setTimeout(this.decodeOnce.bind(this, resolve, reject), this.timeBetweenScansMillis);
+    private decodeOnceWithDelay(resolve: (result: Result) => any, reject: (error: any) => any, cropFn?: CropFn): void {
+        this.timeoutHandler = window.setTimeout(this.decodeOnce.bind(this, resolve, reject, undefined, undefined, cropFn), this.timeBetweenScansMillis);
     }
 
-    private decodeOnce(resolve: (result: Result) => any, reject: (error: any) => any, retryIfNotFound: boolean = true, retryIfChecksumOrFormatError: boolean = true): void {
+    private decodeOnce(resolve: (result: Result) => any, reject: (error: any) => any, retryIfNotFound: boolean = true, retryIfChecksumOrFormatError: boolean = true, cropFn?: CropFn): void {
```

Comment:
My only problem with this cropping is the large ammount of callbacks needed to achieve it, can't we do the crop with less callbacks? 🙏